packages/headless/src/app/engine.ts

```diff
@@ -177,34 +177,31 @@ export function buildEngine<
   const engine = buildCoreEngine(options, thunkExtraArguments);
   const {accessToken, organizationId} = options.configuration;
   const {organizationEndpoints} = options.configuration;
-  let {platformUrl} = options.configuration;
+  const platformUrl = organizationEndpoints?.platform
+    ? organizationEndpoints.platform
+    : options.configuration.platformUrl!;
```

Comment:
```suggestion
  const platformUrl = organizationEndpoints?.platform || options.configuration.platformUrl!
```
Or 
```suggestion
  const platformUrl = organizationEndpoints?.platform ?? options.configuration.platformUrl!
```

Reply:
Thanks Louis. I implemented the suggestion but without the `!` at end as I noticed 
- it does not exist in master - I must have added in inadvertently during refactoring
- there is schema validation down that would catch it, so it is redundant